play/PlayerRules/CharacterCreation/03_Feats.txt

```diff
@@ -42,227 +42,226 @@ Acquired: . . . Who Needs Nano-Technologies?
--5% to Skill checks in combat.
+(Requirement for Expert Melee Training) 
+-1 miss chance in melee. -5% penalty to Skill Checks in combat.
 
 Combat Expertise [Support]: 
 Prerequisites:
-+15% to skill checks for class-related tasks. (if your class is medicine, this 
-wouldn''t apply to hacking, etc.). +5% chance to miss with Guns or Melee in 
-combat.
++20% bonus to skill checks for class-related tasks in combat. (If you are a medic, this 
+would apply to medicine but it wouldn't apply to hacking, etc.). +1 miss chance
+with Guns or Melee in combat.
 
 Concussive Maintenance:
 Prerequisites: 20 skill in repair or Engineer class. STR 4
-For 1 action, you may whack something broken to try to fix it. +20% to 
-repair check or take an action to smack something that you''ve failed to 
-repair. +20% to repair check. Regardless, you can only use concussive 
-maintenance on a target once per encounter.
++20% to repair skill. For 1 action, you may whack something broken to 
+try to fix it. You can use Concussive Maintenance even after you've already failed
+to repair something. You can use concussive maintenance once per target, per encounter.
 
 Conceal Weapon: 
 Prerequisites:
-Make a stealth check to hide your chosen pistol, shotgun, or smg on your 
-person. Minimum score of 20. Roll determines DC for enemy to actively search 
-and find weapon.
+Make a stealth check to hide your chosen pistol, shotgun, or SMG on your 
+person. To succeed, you need a minimum score of 20. Roll determines DC for 
+enemy to actively search and find your hidden weapon.
 
-Cross-Train:
-Prerequisites: Int 8
-Instead of taking your regular Class feat on the next level, you may learn one 
-from another class.
+NOT USED:
+	Cross-Train:
+	Prerequisites: Int 8
+	Instead of taking your regular Class feat on the next level, you may learn one 
+	from another class.
 
 Double Tap: 
-Prerequisites:
-Whenever your hit falls into extra-crit range, fire another bullet as a free
-action. The extra bullet does not proc any other effects and cannot be used to
-double tap again. Only usable with Semi-Auto weapons. Not usable with
-SA Revolvers, Bolt Action, Pump-Action, Full-Auto weapons, etc.
+Prerequisites: 
+Whenever your hit falls into crit range, with a max crit range of 3*, fire another bullet as a
+free action, foregoing the normal bonus-damage from a crit. The extra bullet automatically hits the 
+target if your first bullet does, does not trigger any other effects, and cannot be used to double tap
+again. Only usable with Semi-Auto weapons. Not usable with Single-Action Revolvers, Bolt Action, Pump-Action, 
+Full-Auto weapons, etc.
+	*If you have a crit range greater than or equal to 3, only rolls of 8 or more will
+trigger Double Tap. One exception: the max crit range of 3 can be increased to 4 by the 
+feat "Never Tell Me the Odds". If you have a crit range of less than 3, then crit range applies
+as normal. 

```

Comment:
I like this! I especially like that we're merging things with anti-Lovelies-esque breaking patches thought out beforehand!